sopel/__init__.py

```diff
@@ -62,9 +67,13 @@ def _version_info(version=__version__):
         level = 'final'
     else:
         level = 'alpha'
-    version_type = namedtuple('version_info',
+
+    # this overlaps with a name from the outer scope, true,
+    # but the alternative was renaming the type to `version_type`
+    # -- not preferable
+    version_info = namedtuple('version_info',
                               'major, minor, micro, releaselevel, serial')
```

Comment:
The class definition is not public (because it is within a function) so I can't even be used as a type annotation, so you can change it to whatever you want.

And yes, "WhateverYouWant" is a valid name.

Reply:
Have added a fixup commit to change this, and tell my future self how to reword the original commit message.